packages/grafana-data/src/transformations/transformers/joinDataFrames.ts

```diff
@@ -34,7 +34,7 @@ export function pickBestJoinField(data: DataFrame[]): FieldMatcher {
 }
 
 /**
- * @alpha
+ * @internal
```

Comment:
Do we have documentation regarding these tags (i.e. definition / when to use them?) - in this case does marking this interface as `internal` vs `alpha` imply that users of @grafana/data shouldn't import this interface / use it in their own code? Do these tags trigger warnings if that is the case?